### PR TITLE
Update download URLs

### PR DESCRIPTION
**Pull request by tashian**

### CODE REVIEW

File: step-ca/installation.mdx [hunk 1]
```diff
@@ -84,14 +84,14 @@ To uninstall, run `scoop uninstall`, then remove the configuration directory `$H
...
-    wget https://dl.step.sm/gh-release/cli/docs-ca-install/v0.23.2/step-cli_0.23.2_amd64.deb
+    wget https://dl.smallstep.com/gh-release/cli/docs-ca-install/v0.23.2/step-cli_0.23.2_amd64.deb
     sudo dpkg -i step-cli_0.23.2_amd64.deb
     ```
 
 To install `step-ca`, download and install the Debian package for your platform from our [latest release](https://github.com/smallstep/certificates/releases/latest):
 
     ```shell
-    wget https://dl.step.sm/gh-release/certificates/docs-ca-install/v0.23.2/step-ca_0.23.2_amd64.deb
+    wget https://dl.smallstep.com/gh-release/certificates/docs-ca-install/v0.23.2/step-ca_0.23.2_amd64.deb
```
Comment (andysmallstep) [inline step-ca/installation.mdx:94]: This one gave me a 404

File: step-ca/installation.mdx [hunk 2]
```diff
@@ -122,14 +122,14 @@ Big shout out to the maintainers of these packages! We appreciate you.
...
-    wget https://dl.step.sm/gh-release/cli/docs-ca-install/v0.23.2/step-cli_0.23.2_amd64.rpm
+    wget https://dl.smallstep.com/gh-release/cli/docs-ca-install/v0.23.2/step-cli_0.23.2_amd64.rpm
     sudo dnf install step-cli_0.23.2_amd64.rpm
     ```
 
 To install `step-ca`, download and install the RPM package for your platform from our [latest release](https://github.com/smallstep/certificates/releases/latest):
 
     ```shell
-    wget https://dl.step.sm/gh-release/certificates/docs-ca-install/v0.23.2/step-ca_0.23.2_amd64.rpm
+    wget https://dl.smallstep.com/gh-release/certificates/docs-ca-install/v0.23.2/step-ca_0.23.2_amd64.rpm
```
Comment (andysmallstep) [inline step-ca/installation.mdx:132]: This one gave me a 404

File: step-ca/installation.mdx [hunk 3]
```diff
@@ -201,7 +201,7 @@ To download and install the `step` and `step-ca` binaries on an amd64 machine:
     Download and install the Linux tarball from our [latest release](https://github.com/smallstep/certificates/releases/latest):
 
     ```shell
-    curl -L https://dl.step.sm/gh-release/certificates/docs-ca-install/v0.23.2/step-ca_linux_0.23.2_amd64.tar.gz \
+    curl -L https://dl.smallstep.com/gh-release/certificates/docs-ca-install/v0.23.2/step-ca_linux_0.23.2_amd64.tar.gz \
```
Comment (andysmallstep) [inline step-ca/installation.mdx:204]: This one gave me a 404

**Review by andysmallstep — COMMENTED** [review]
Hi, @tashian - Just a few 404s on some of the URLs that I couldn't resolve as suggestions.

**Comment by tashian** [comment]
@dopey should we still merge this?

**Comment by dopey** [comment]
Yep, this is still good for merge. All those urls should work.

**Review by andysmallstep — APPROVED** [review]

**Pull request merged**